pkgs/development/libraries/qt-5/5.8/qtbase/default.nix

```diff
@@ -15,7 +17,8 @@
   cups ? null, mysql ? null, postgresql ? null,
 
   # options
-  mesaSupported, mesa,
+  mesaSupported ? (!stdenv.isDarwin),
```

Comment:
Why would the default value have any difference when `mesaSupported` is defined in `all-packages.nix`? And why would you want to disable mesa on Darwin?